toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -119,14 +119,12 @@ os:
       - rd.luks=0
       - rd.hostonly=0
       - ipe.enforce=0
-      - systemd.journald.forward_to_console=1
 
   packages:
     remove:
```

Comment:
Just merged package cleanup into steamboat, consider picking up.

Reply:
Updates pulled in